types/node/http.d.ts

```diff
@@ -159,6 +159,7 @@ declare module "http" {
 
         constructor(url: string | URL | ClientRequestArgs, cb?: (res: IncomingMessage) => void);
 
+        path: string;
```

Comment:
I think here should be `readonly path: string;`

Reply:
Thank you for your suggestion. I've updated them.